### ISSUE TITLE
[Feature Request]: async blob submissions

### ISSUE THREAD

**Issue by Bidon15** [celestiaorg/celestia-node#3296]
### Implementation ideas

the api should have a way to let users submit async blob submission calls
should be delivered in tandem with https://github.com/celestiaorg/celestia-app/issues/5672

ref: https://github.com/celestiaorg/celestia-node/issues/3256

**Comment by 0xRampey** [comment]
Any plans to return result statuses through a channel on async blob submission? Like `INCLUDED`, `FINALIZED`, `CANCELLED` etc

**Comment by renaynay** [comment]
@0xRampey yes something like that, we are considering adding an endpoint to blob module that would be able to submit a tx and query for its status, returning the result through a channel once included.

**Comment by Wondertan** [comment]
The current Signer integration doesn't make blob submission async. Yes, it changed the tx submission mode from Block to Sync, but [this line](https://github.com/celestiaorg/celestia-app/blob/171a54769bd8b87e60b3d7107f3fc050767c3549/pkg/user/signer.go#L197) makes it equivalent to what we had before
